test/asg/Main.hs

```diff
@@ -69,25 +76,15 @@ import Covenant.Test
     tyAppTestDatatypes,
     typeIdTest,
   )
-import Covenant.Type
-  ( AbstractTy,
-    BuiltinFlatT (UnitT),
-    CompT (Comp0, Comp1, Comp2, CompN),
-    CompTBody (ArgsAndResult, ReturnT, (:--:>)),
-    ValT (BuiltinFlat, Datatype, ThunkT),
-    arity,
-    boolT,
-    byteStringT,
-    integerT,
-    tyvar,
-  )
+import Covenant.Type (AbstractTy, BuiltinFlatT (IntegerT, UnitT), CompT (Comp0, Comp1, Comp2, CompN), CompTBody (ArgsAndResult, ReturnT, (:--:>)), ValT (BuiltinFlat, Datatype, ThunkT), arity, boolT, byteStringT, integerT, tyvar)
 import Covenant.Util (pattern ConsV, pattern NilV)
 import Data.Coerce (coerce)
 import Data.Kind (Type)
 import Data.Map qualified as M
 import Data.Maybe (fromJust)
 import Data.Vector qualified as Vector
-import Data.Wedge (Wedge (Here, Nowhere), wedgeLeft)
+import Data.Wedge (Wedge (Here, Nowhere, There), wedgeLeft)
+import Debug.Trace (traceM)
```

Comment:
Is there still tracing code left in somewhere?

Reply:
Oops. Gone now.